meta-balena-common/recipes-support/ca-certificates/ca-certificates/extract-balena-ca

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+CONFIG_JSON="/mnt/boot/config.json"
+CA_BUNDLE="/etc/ssl/certs/ca-certificates.crt"
```

Comment:
CA_BUNDLE is unused? Good habit to run shellcheck on shell scripts :)

Reply:
Damn I did changes after my "final" check :) Will remove that.